dagger/run_dbt_spark_tests.py

```diff
@@ -16,7 +16,7 @@
     if env_name.startswith(("DD_", "DBT_"))
 }
 
-TESTING_ENV_VARS.update({"ODBC_DRIVER": "Simba"})
+TESTING_ENV_VARS.update({"ODBC_DRIVER": "/opt/simba/spark/lib/64/libsparkodbc_sb64.so"})
```

Comment:
This will fail on the Windows CI machine though, right? Or are we only running on MacOS/Ubuntu?

Reply:
Specifically the container OS is determined here: https://github.com/dbt-labs/dbt-spark/pull/971/files#diff-ca1b9f8fafcab3a8c6df7e520e725035642f58e54df0d88402a7a61fac5578b6L95